specification/common-types/resource-management/v6/managedidentitywithdelegation.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Common types",
+    "version": "6.0"
+  },
+  "paths": {},
+  "definitions": {
+    "DelegatedResource": {
+      "type": "object",
+      "description": "Delegated resource properties - internal use only.",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The ARM resource id of the delegated resource - internal use only."
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The tenant id of the delegated resource - internal use only."
+        },
+        "referralResource": {
+          "type": "string",
+          "description": "The delegation id of the referral delegation (optional) - internal use only."
```

Comment:
>delegation [](http://example.com/codeflow?start=30&length=10)

what kind of id format is this?

Reply:
same as the other, we are migrating what we currently have on v5: https://github.com/Azure/azure-rest-api-specs/blob/0bbf10474eb460cb35d7e2d3524054418eb19406/specification/common-types/resource-management/v5/managedidentitywithdelegation.json#L22

Reply:
Maybe its a case of being deliberately vague?
I could find no clues for this in the azure rest api specs repo - perhaps tellingly - not a single example!
Is this really only used in internal APIs?